docs/components/content/ComponentCard.vue

```diff
@@ -124,7 +124,7 @@ const meta = await fetchComponentMeta(name)
 
 // Computed
 
-const fullProps = computed(() => ({ ...baseProps, ...componentProps }))
+const fullProps = computed(() => ({ ...componentProps, ...baseProps }))
```

Comment:
Shouldn't it be cleaner to remove `baseProps` completely? Its usage was to hide some props from the generated code on purpose.

Reply:
With such simple modifications, we don't need to modify the all MD file anymore?

Reply:
And, if we usage `basePrpos`, its can hide some controlTree